.github/workflows/ci.yml

```diff
@@ -69,6 +67,6 @@ jobs:
 
       - name: Push Images to ECR
         run: |
-          .github/retag-and-push.sh navigator-frontend latest
+          .github/retag-and-push.sh navigator-frontend-${{ matrix.theme }} latest
```

Comment:
I believe this will require manually setting up an ecr repo for each new theme, just an fyi

Reply:
Have done this, thanks!!